vite-project/src/pages/FolderPage.jsx

```diff
@@ -0,0 +1,73 @@
+import AddLink from "../components/AddLink";
+import SearchBar from "../components/SearchBar";
+import CardList from "../components/CardList";
+import CardListFolderMenu from "../components/CardListFolderMenu";
+import CardListTitleMenu from "../components/CardListTitleMenu";
+import { useState, useEffect } from "react";
+import { FOLDER_PAGE_API_URL } from "../constants/constant";
+
+function FolderPage() {
+  const [cardListMenuData, setCardListMenuData] = useState({});
+  const [cardLinkData, setCardLinkData] = useState({});
+  const [selectedFolderName, setSelectedFolderName] = useState("");
+
+  async function getFolderPageMenuData() {
+    try {
+      const response = await fetch(`${FOLDER_PAGE_API_URL}/1/folders`);
+      const linkData = await response.json();
+      setCardListMenuData(linkData);
```

Comment:
요청하는 API endpoint를 봤을 때, 특정 유저의 폴더 목록을 요청하는 것인데, 이름이 다르게 변하고 있어서(linkData -> cardLinkMenuData) 혼동을 줄 수 있을거같아 제안드립니다.
```suggestion
      const response = await fetch(`${FOLDER_PAGE_API_URL}/1/folders`);
      const { data : folders } = await response.json();
      setFolders(folders);
```